src/js/contexts/AnnounceContext/announce-context.stories.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { grommet } from 'grommet/themes';
+import { Box, Grommet, Text, AnnounceContext } from 'grommet';
+
+class Announcer extends Component {
+  static propTypes = {
+    announce: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    const { message, role, politenessSetting } = this.props;
+    this.state = {
+      message:
+        message || 'Here is a simple announcement. This will soon disappear',
+      politenessSetting: politenessSetting || 'polite',
+      role: role || 'log',
+    };
+  }
+
+  componentDidMount() {
+    const { announce } = this.props;
+    const { message, politenessSetting } = this.state;
+    const timeout = 3000;
+    announce(message, politenessSetting, timeout);
```

Comment:
Can we update the AnnounceContext/doc.js file to mention the arguments to the `announce()` function?

Reply:
Addressed in recent commit. Thanks !!